spec/schemas/flow_framework._common.yaml

```diff
@@ -0,0 +1,100 @@
+openapi: 3.1.0
+info:
+  title: Schemas of flow_framework._common category
+  description: Schemas of flow_framework._common category
+  version: 1.0.0
+paths: {}
+components:
+  schemas:
+    FlowFrameworkException:
```

Comment:
Exception vs. Error below, is that intentional?

I think this sets a nice precedent for specialized errors, should we have a `flow_framework._errors.yaml`?

Reply:
I will change it to `FlowFrameworkAPIDisabledError` for consistency.

Also will change the file name to `flow_framework._errors.yaml`